### DISCUSSION TITLE
Questions , Concerns, Suggestions?

### DISCUSSION THREAD

**Discussion by jessel92** [The-Mad-Noodle/Mad-Noodle-Via-Support#2] [General]
### Just a general thread for any questions you may have or suggestions you can think of! 
Whether is be setup or usage, maybe this can be a place for help!

**Comment by nickv2002** [comment]
I just picked up a NoodePad (v1) device secondhand and I want to reset it to some sort of "default" rather than reprograming every key in Via. Is there some default keymap I could download and apply?

**Reply by jessel92** [reply]
Hey Nick! 

Sorry for the little bit of a late reply! 
But yeah! If you download the `themadnoodle_noodlepad_default.hex` from [THIS](https://github.com/The-Mad-Noodle/The-Mad-Noodle-Keypads-Gen2/releases/tag/1.1) link, that would be the firmware that would've came by default on the Noodlepad V1 back then. 

But QMK has changed a lot since then, so there is a more recent "Default" firmware file (`themadnoodle_noodlepad_v1_default.hex`)
 available [HERE](https://github.com/The-Mad-Noodle/QMK-Mad-Noodle-KeyPads/releases) , which i'd probably recommend using since it's basically the same, just updated. 
 
And since QMK has changed so much, if you're going to try to mess with QMK at all, make sure to use the source files from that same most recent https://github.com/The-Mad-Noodle/QMK-Mad-Noodle-KeyPads repo

Hope this helps! 
Best,
Jesse
The Mad Noodle

**Comment by Julien1988** [comment]
Hi, I bought the UDON:13 V2, nice quality and pleasant to use.

As I move a lot of space my setup I spend a lot of time plugging and unplugging the cables.

After information I saw this possibility to add a bt adapter and external battery on this site: https://handheldsci.com/kb/

Do you think it would be compatible with the UDON?

Have a good day. :)
